GPflow/model.py

```diff
@@ -147,6 +152,7 @@ def _compile(self, optimizer=None):
         sys.stdout.flush()
 
         self._feed_dict_keys = self.get_feed_dict_keys()
+
```

Comment:
Not a big deal but try to avoid random whitespace. Can make it hard to see changes. No need to revert.